airbyte-integrations/connectors/source-sftp-bulk/setup.py

```diff
@@ -13,7 +13,7 @@
     "pandas==1.5.0",
 ]
 
-TEST_REQUIREMENTS = ["pytest~=6.1", "connector-acceptance-test", "docker==5.0.3"]
+TEST_REQUIREMENTS = ["pytest~=6.1", "connector-acceptance-test", "docker==5.0.3", "pytest-docker"]
```

Comment:
This pytest plugin exposes a useful `docker_ip` fixture which resolve the correct current docker host IP.
I preferred to install it as a dep instead of copy pasting the fixture code...